contracts/mainnet/connectors/uniswapV3/main.sol

```diff
@@ -0,0 +1,145 @@
+pragma solidity =0.7.6;
+pragma abicoder v2;
+
+/**
+ * @title Uniswap v3.
+ * @dev Decentralized Exchange.
+ */
+
+import {TokenInterface} from "../../common/interfaces.sol";
+import {Helpers} from "./helpers.sol";
+import {Events} from "./events.sol";
+
+abstract contract UniswapResolver is Helpers, Events {
```

Comment:
@cryptoDev222 I think we need a seperate function for `collect()?`

Reply:
Okay, got it.